moveit_planners/trajopt/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(moveit_planners_trajopt)
+
+# At least C++11 required for TrajOpt
```

Comment:
remove this comment, we don't know if that's true. and 11 doesn't match 14

Reply:
done